prediction_market_agent_tooling/markets/seer/seer.py

```
@@ -281,14 +295,57 @@
+
+    @tenacity.retry(
+        stop=tenacity.stop_after_attempt(3),
+        wait=tenacity.wait_fixed(1),
+        after=lambda x: logger.debug(
+            f"seer_sell_outcome_tx failed, {x.attempt_number=}."
+        ),
+    )
+    def sell_tokens(
+        self,
+        outcome: bool,
+        amount: TokenAmount,
```

Ruff (0.8.2)

Undefined name `TokenAmount`

(F821)